test/e2e/fixture/scylla/zonal.scyllacluster.yaml.tmpl

```diff
@@ -0,0 +1,38 @@
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+metadata:
+  generateName: basic-zonal-
```

Comment:
nit: I'd call it just `basic` so the names are consistent when debugging

Reply:
done